### ISSUE TITLE
Warning message: This build has been signed with public keys

### ISSUE THREAD

**Issue by MrWolf-to** [topjohnwu/Magisk#5324]
Device: Xiaomi Redmi 5 Plus
Android version: 8.1.0
Magisk version name: 24.1
Magisk version code: 24101

Hi, I have a Xiaomi Redmi 5 Plus with Resurrection Remix OS RR-O-v6.2.1-20181003-vince-Final (Android 8.1.0)
I've just updated to Magisk 24.1 from 23.0 and now, when I boot the phone, I get this warning message from the Trust Platform:
This build has been signed with public keys

I've never seen this message before. I've googled a bit, but I can't understand well what it means and why it has appeared only know, and, most of all, if it can be safely ignored.
Thanks.

**Comment by vvb2060** [comment]
Ask the ROM developer

**Comment by MrWolf-to** [comment]
Ok... Since this message has appeared only after I updated Magisk, I thought it was related to Magisk.

**Comment by adonios77** [comment]
Device: Xiaomi Mi A2 lite
Android version: 11
OS: LineageOS 18.1
Magisk version: 24.3

Red exclamation message: 
/!\ Trust · System security
This build was signed with public keys

**Comment by MrWolf-to** [comment]
Thanks for your answer. I've already done just as you have written: disabled that notification.
The only thing I couldn't understand is why it appeared only after I updated Magisk.

**Comment by adonios77** [comment]
> The only thing I couldn't understand is why it appeared only after I updated Magisk.

I think that Magisk's update was a coincidence.
As explained above, it's possibly caused when **Custom ROM** builds based on "**public key**".
